libr/io/p/io_ar.c

```diff
@@ -94,6 +94,7 @@ RIOPlugin r_io_plugin_ar = {
 	.name = "ar",
 	.desc = "Open ar/lib files [ar|lib]://[file//path]",
 	.license = "LGPL3",
+	.prefixes = "ar://, lib://",
```

Comment:
what about making this field an actual array of char* instead of a plain string?

Reply:
char ** type cannot be directly assigned like `char **a = {"one", "two", NULL}`. One way is using `char name[]` but even that requires using `strcpy` if we are assigning after declaration. 
Also since it has no other use other than directly displaying, is it really required?

Reply:
Sorry I didn't see that comment, ideally the json should yield a list for the available prefixes, so it's actually parsable :)
Something like:
```
{"IO_Plugins":[{"Permissions":"rw_","Name":"ar","Description":"Open ar/lib files [ar|lib]://[file//path]","License":"LGPL3","prefixex": ["ar://", "lib://"]}....}
```

Reply:
Okay xarxes, I will make the change and push them by tomorrow.